layouts/partials/top.html

```diff
@@ -1,6 +1,6 @@
 <div id="top">
   <div id="logo">
-    <img src="/css/logo.svg" width="125px" alt="logo" />
+    <img src="/css/logo.svg" width="125" height="93" alt="logo" />
```

Comment:
Pixelwerte sind nur in CSS möglich. Die width / height Syntax kann nur integer Werte annehmen